tests/zexperimental/test_mcmc.py

```diff
@@ -239,6 +239,33 @@ def test_RegularizedLinearRTO_regression_warmup(target: cuqi.density.Density):
     sampler_new = cuqi.experimental.mcmc.RegularizedLinearRTONew(target, stepsize=1e-3)
     assert_true_if_warmup_is_equivalent(sampler_old, sampler_new)
 
+def create_lmrf_prior_target(dim=16):
```

Comment:
After rebasing on main there might be another place to add the sampler. The place related to state.

Reply:
I also added `UGLANew` to `state_history_targets`